inference_segmentation.py

```diff
@@ -158,19 +158,21 @@ def segmentation(param,
 
     """
     sample = {"image": None, "mask": None, 'metadata': None}
-    start_seg = time.time()
+    # start_seg = time.time()
```

Comment:
Do we need to keep this? (same for the end time)

Reply:
I will clean it out